python/lsst/pex/config/config.py

```diff
@@ -237,23 +237,26 @@ def save(self, outfile, instance):
         """
         value = self.__get__(instance)
         fullname = _joinNamePath(instance._name, self.name)
+
+        # write full documentation string as comment lines (i.e. first character is #)
+        doc = "# " + re.sub(r'\n', r'\n# ', str(self.doc))
```

Comment:
I don't think you actually need `re` here -- this should be equivalent:
```
doc = "# " + self.doc.replace("\n", "\n# ")
```
Saves an import, marginally less verbose, but it's probably not a big deal.